src/services/onboard.ts

```diff
@@ -23,10 +23,10 @@ export const createOnboard = (chainConfigs: ChainInfo[]): OnboardAPI => {
     id: hexValue(parseInt(cfg.chainId)),
     label: cfg.chainName,
     rpcUrl: getRpcServiceUrl(cfg.rpcUri),
-    publicRpcUrl: cfg.publicRpcUri.value,
```

Comment:
Good catch. How did you find out about this being the problem?
I tried to look for github issues which explain that public RPC is broken for hw wallets but didnt find anything.

Reply:
I copied hardcoded chain configs from their examples first, and they worked. The only difference was this.

Reply:
Not sure what you mean, Aaron. Feel free to try.

Reply:
Sorry, I meant `rpcUrl` and `publicRpcUrl` as the same time. It doesn't work though.